src/Compilers/CSharp/Portable/CSharpResources.resx

```diff
@@ -1497,7 +1497,7 @@ If such a class is used as a base class and if the deriving class defines a dest
     <value>'{0}' in explicit interface declaration is not an interface</value>
   </data>
   <data name="ERR_InterfaceMemberNotFound" xml:space="preserve">
-    <value>'{0}' in explicit interface declaration is not a member of interface</value>
+    <value>'{0}' in explicit interface declaration is not found among members of the interface that can be implemented</value>
   </data>
```

Comment:
This feels like a regression in the quality of the diagnostic. It would be nice to know *why* it wasn't "found among the members that can be implemented". Is there no such member? Is it inaccessible?